src/main/kotlin/io/github/oxisto/reticulated/Pair.kt

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) 2020, Fraunhofer AISEC. All rights reserved.
```

Comment:
I suggest the following copyright header:
```
* Copyright (c) 2020, Christian Banse and Andreas Hager. All rights reserved.
```

Reply:
Done commit e01b9e2189b76694cb29ff42737780b701664745